flink-staging/flink-ml/src/test/resources/LabeledSequenceData

```diff
@@ -0,0 +1,118 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+1,2,2,0,2,0,1,0,0,2,0,0:2,1,0,1,2,2,0,0,1,0,2,2
```

Comment:
I have written my own code. I hand-crafted the initial probability vector, the transition matrix and the probability matrix. And I executed the HMM based on the two matrices to generate data